lib/lexer_mdx.mll

```diff
@@ -89,44 +65,35 @@ and cram_text section = parse
         newline lexbuf;
         `Section section :: cram_text (Some section) lexbuf }
   | "  " ([^'\n']* as first_line) eol
-      { let header = Some (Block.Header.Shell `Sh) in
+      { let start = Lexing.lexeme_start_p lexbuf in
+        newline lexbuf;
+        let header = "sh" in
         let requires_empty_line, contents = cram_block lexbuf in
         let contents = first_line :: contents in
-        let labels = [] in
-        let legacy_labels = false in
-        let loc = Location.curr lexbuf in
-        List.iter (fun _ -> newline lexbuf) contents;
+        let label_cmt = Some "" in
```

Comment:
Shouldn't this be `None`?

Reply:
No, this is a direct translation of what was done before (i.e. no labels, and legacy_labels is `false`). This basically encode that there were no label without triggering any potential warnings for using the old syntax!